src/ui/components/ServerLaunchSection.tsx

```diff
@@ -277,30 +277,34 @@ export default class ServerLaunchSection extends React.Component<IProps, IState>
         {this.state.grasscutterEnabled && (
           <div>
             <div className="ServerConfig" id="serverConfigContainer">
-              <TextInput
-                id="ip"
-                key="ip"
-                placeholder={this.state.ipPlaceholder}
-                onChange={this.setIp}
-                initalValue={this.state.ip}
-              />
-              <TextInput
-                style={{
-                  width: '10%',
-                }}
-                id="port"
-                key="port"
-                placeholder={this.state.portPlaceholder}
-                onChange={this.setPort}
-                initalValue={this.state.port}
-              />
-              <HelpButton contents={this.state.portHelpText} />
-              <Checkbox
-                id="httpsEnable"
-                label={this.state.httpsLabel}
-                onChange={this.toggleHttps}
-                checked={this.state.httpsEnabled}
-              />
+              <div id="ServerConfigCheckbox">
```

Comment:
These name changes are technically breaking changes for themes (if they use the old selectors). Please update `THEMES.md` docs please!

Reply:
no need to update this since I removed the div changes